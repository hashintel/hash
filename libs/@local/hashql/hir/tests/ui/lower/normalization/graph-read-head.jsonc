//@ run: pass
//@ description: todo
[
  "::graph::tail::collect",
  ["::graph::head::entities", ["::graph::tmp::decision_time_now"]]
]
