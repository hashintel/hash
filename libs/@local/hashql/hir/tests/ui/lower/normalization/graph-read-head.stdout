════ Initial HIR ═══════════════════════════════════════════════════════════════

::graph::tail::collect(::graph::head::entities(::graph::tmp::decision_time_now()))

════ HIR after normalization ═══════════════════════════════════════════════════

#let %0 = ::graph::tmp::decision_time_now()
     %1 = ::graph::head::entities(%0) |> ::graph::tail::collect
in
%1
